sdk/textanalytics/azure-ai-textanalytics/tests/test_detect_language_async.py

```diff
@@ -144,15 +144,14 @@ async def test_input_with_all_errors(self, client):
 
     @GlobalTextAnalyticsAccountPreparer()
     @TextAnalyticsClientPreparer()
-    @pytest.mark.xfail
     async def test_too_many_documents(self, client):
-        # marking as xfail since the service hasn't added this error to this endpoint
-        docs = ["One", "Two", "Three", "Four", "Five", "Six"]
+        docs = [u"hello world"] * 1000
```

Comment:
shouldn't this need to be 1001?

Reply:
With 1001 it gives a different error of "(InvalidDocumentBatch) Batch request contains too many records. Max 1000 records are permitted." which is thrown directly by the service. With 1000, I am able to repro the too many documents error

Reply:
Ohhh got it, so they didn't decrease the batch limit for detect language then. makes sense.

Reply:
I was actually wrong about 1000. That call goes through fine. I thought that it was raising based on how we had a try except in the tests and were validating the exception, but the call never makes its way into the except. Going to add an assert False at the end of each test to make sure the tests are being run correctly